synadm/cli/notice.py

```diff
@@ -135,7 +135,8 @@ def confirm_prompt():
         formatted_content = formatted if formatted else plain_content
 
     if regex:
-        if "users" not in helper.api.user_list(0, 100, True, False, "", ""):
+        if "users" not in helper.api.user_list(0, 100, True, False, "", "",
+                                               None):
```

Comment:
There is another call of user_list in line 97 in this file. I think that also requires this additional argument. Thanks!